pkg/gen/input/makefile/internal/file/makefile_gen_go_mk.go

```diff
@@ -35,8 +35,9 @@ SOURCES        := $(shell find . -name '*.go')
 VERSION        := $(shell architect project version)
 LDFLAGS        ?= -w -linkmode 'auto' -extldflags '-static' \
   -X '$(shell go list .)/pkg/project.buildTimestamp=${BUILDTIMESTAMP}' \
-  -X '$(shell go list .)/pkg/project.gitSHA=${GITSHA1}'
-.DEFAULT_GOAL := build
+  -X '$(shell go list .)/pkg/project.gitSHA=${GITSHA1}' \
+  -X '$(shell go list .)/pkg/project.version=${VERSION}'
```

Comment:
But don't we want this to be set in `version.go`? We don't want this to be overwritten at the compile time.

Reply:
The problem is when the binary is downloaded as a GitHub release. 
In that case there is no version. 

```
apptestctl version
Git Commit:     n/a
Go Version:     go1.15.3
OS / Arch:      darwin / amd64
Source:         https://github.com/giantswarm/apptestctl
```

This is what was fixed in https://github.com/giantswarm/apptestctl/pull/58